File: blind 75 leetcode/Arrays and Strings/contains_duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        return len(nums) != len(set(nums))
    def alternativeSolution(self, nums: List[int]) -> bool:
    	counter = dict()
    	for i in nums:
    		if i in counter:
    			return True
    		else:
    			counter[i] = 1
    	return False
            

File: node_js_ds_and_algos_implementations/arrays and strings/hashmaps/contains_duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    let unique = new Set()
    for (let i = 0; i < nums.length; ++i){
        unique.add(nums[i])
    }
    return unique.size !== nums.length
};